ob1k-core/src/main/java/com/outbrain/ob1k/server/builder/NoOpBuilderProvider.java

```diff
@@ -0,0 +1,16 @@
+package com.outbrain.ob1k.server.builder;
+
+public class NoOpBuilderProvider<T> implements BuilderProvider<T> {
```

Comment:
Oh yeah. Thought I had deleted it. That was the plan at least... oh well, one more push.